cluster-provision/k8s/1.20/manifests/cni.diff

```diff
@@ -1,5 +1,5 @@
---- a/cluster-provision/k8s/1.19/manifests/cni.do-not-change.yaml
```

Comment:
I honestly don't understand this change

Reply:
It seems like this file was copied from older k8s provision without changing the path.
You can see that this `cni.diff` file is in `cluster-provision/k8s/1.20/manifests`, but it's applied to older
`a/cluster-provision/k8s/ => 1.19 <= /manifests/cni.do-not-change.yaml`.

`cni.do-not-change.yaml` hasn't changed between versions, but still, the patch should be applied to the right manifest

Reply:
i see 4 lines changes, instead of 2.

I would have understood something like: 
```
--- a/cluster-provision/k8s/1.19/manifests/cni.do-not-change.yaml
+++ b/cluster-provision/k8s/1.20/manifests/cni.do-not-change.yaml
```

I see instead: 
```
--- a/cluster-provision/k8s/1.19/manifests/cni.do-not-change.yaml
+++ b/cluster-provision/k8s/1.19/manifests/cni.do-not-change.yaml
--- a/cluster-provision/k8s/1.20/manifests/cni.do-not-change.yaml
+++ b/cluster-provision/k8s/1.20/manifests/cni.do-not-change.yaml
```

Am I missing something ?

Reply:
diff of a diff once again.

Reply:
one thing that just occurred to me: instead of pointing to the "correct" copy (all files have the same contents) you could just delete them, and refer to the 1.19 configuration. Until there's a need for a different config, moment on which we'd have to copy the correct config.